app/assets/admin/js/settings.js

```diff
@@ -979,6 +989,25 @@ jQuery( document ).ready( function ( $ ) {
 		window.location.href = '/wp-admin/tools.php?page=remove_personal_data';
 	});
 
+	/**
+	 * Show error message if validation fails.
+	 *
+	 * @param selector
+	 * @param error_msg
+	 * @returns {boolean}
```

Comment:
Blank line before `return` doc comment. Also add type before variable name and add its description.
`@param string error_msg An error message to display`

Reply:
Please address this review.